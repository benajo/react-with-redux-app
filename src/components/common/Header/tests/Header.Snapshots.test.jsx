import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Header from '../Header';

it('render Header component', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Header />
    </MemoryRouter>,
  );
  expect(tree).toMatchSnapshot();
});
