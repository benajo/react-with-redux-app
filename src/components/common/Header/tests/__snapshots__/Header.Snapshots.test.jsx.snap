// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Header component 1`] = `
<nav>
  <a
    aria-current="page"
    className="active"
    href="/"
    onClick={[Function]}
    style={
      Object {
        "color": "#F15B2A",
      }
    }
  >
    Home
  </a>
   | 
  <a
    aria-current={null}
    href="/courses"
    onClick={[Function]}
  >
    Courses
  </a>
   | 
  <a
    aria-current={null}
    href="/authors"
    onClick={[Function]}
  >
    Authors
  </a>
   | 
  <a
    aria-current={null}
    href="/about"
    onClick={[Function]}
  >
    About
  </a>
</nav>
`;
